windows/InstallMode.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace CodePush.ReactNative
+{
+    enum InstallMode
+    {
+        IMMEDIATE = 0,
```

Comment:
These names should be pascal-cased, and you can omit the values, since the compiler will take care of it.